.gitignore

```diff
@@ -58,3 +59,7 @@ dist-node
 browsertest/package.json
 browsertest/browsertest.js
 browsertest/browsertest.min.js
+
+# lock files (required version is npm below 3)
```

Comment:
Do we really want to add these? We want to be discouraging people from creating them in the first place.

Reply:
if user accidentally use npm >= 5, it will automatically adds lock file.
Therefore, I thought it will be good to added to ignorefile until we can support higher version

Reply:
One thought is whether having to fight with git to keep lockfiles around might encourage people to use the package manager we support to avoid that struggle?

Reply:
wouldn't it increase the PR discussion every time someone commits the lock file?

Reply:
Probably. Let's add the here then, but text should be more like `Supported package manager is npm 3`.